browser/brave_profile_prefs.cc

```diff
@@ -61,6 +62,11 @@
 #include "brave/components/gemini/browser/pref_names.h"
 #endif
 
+#if BUILDFLAG(BITCOIN_DOT_COM_ENABLED)
+#include "brave/components/moonpay/browser/moonpay_pref_utils.h"
+#include "brave/components/moonpay/common/moonpay_pref_names.h"
```

Comment:
nit - the convention for this is just pref_names.h/pref_names.cc

Reply:
these should be using the MOONPAY_ENABLED buildflag to match what is in gn. You must always use the same buildflags in c++/gn

Reply:
^^ applies to all moonpay files

Reply:
Now just using `MOONPAY_ENABLED` buildflag, updated pref file names